src/components/CurrencyInputPanel/index.tsx

```diff
@@ -166,33 +179,35 @@ interface CurrencyInputPanelProps {
   error?: boolean
   maxLength?: number
   filterWrap?: boolean
+  lockIcon?: boolean
+}
+
+const noop = (value: string) => {
```

Comment:
I see many `noop` like this in source code, can you just modify component `NumericalInput` to accept undefined value.